Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -118,6 +120,11 @@ screen initialState =
             -- push NewUser -- TODO :: Handle the functionality
             _ <- if initialState.data.config.enableMockLocation then isMockLocation push IsMockLocation else pure unit
             _ <- launchAff $ flowRunner defaultGlobalState $ checkForLatLongInSavedLocations push UpdateSavedLoc initialState
+            if(initialState.props.currentStage /= HomeScreen) then do
+              runEffectFn4 setMapPadding 0 0 0 0
```

Comment:
what will happen after ride assigned? route will go behind the bottom sheet ?

Reply:
No it won't affect, we are already setting map height to `(V (((screenHeight unit)/ 15)*10))` in `[RideAccepted, RideStarted, ChatWithDriver]` these stages